iOS/BattleBuddy/Source/View/View Controllers/Learn/PriceCheckViewController.swift

```diff
@@ -82,6 +82,18 @@ extension PriceCheckViewController: UISearchBarDelegate {
     func searchBar(_ searchBar: UISearchBar, textDidChange searchText: String) {
         controller.updateSearchText(searchText)
     }
+
+    func searchBarSearchButtonClicked(_ searchBar: UISearchBar) {
+        searchBar.resignFirstResponder()
+    }
+}
+
+// MARK: - UIScrollViewDelegate
+
+extension PriceCheckViewController {
+    override func scrollViewDidScroll(_ scrollView: UIScrollView) {
+        searchBar.resignFirstResponder()
+    }
```

Comment:
Perhaps we can replace this with ```scrollViewWillBeginDragging``` instead, as that will be called once upon scrolling as opposed to ```scrollViewDidScroll``` which is called a shitload during scrolling? Thoughts? Thanks for the PR!

Reply:
Also would want to re-target this PR to the `release/1.3.0` branch if possible!